tests/cli/BUILD

```diff
@@ -14,6 +14,7 @@ ts_test_suite(
     data = [
         ":node_modules",
         "//packages/@dataform/core:package_tar",
+        "//sandbox/vm:worker_bundle",
```

Comment:
This makes your tests pass, but the CLI itself isn't going to actually work, plz remove!

Reply:
And see my other comment for fix